### PR TITLE
feature/PEER-786 Set the languages and autotranslations for a single community

### PR DESCRIPTION
**Pull request by AweAwe95**

### CODE REVIEW

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+const DropdownTrigger: React.FC<DropdownTriggerProps> = ({
+  selectedLanguages,
+  allLanguages,
+  removeSelectedLanguage,
+  placeholder,
+}): JSX.Element => {
+  const [isOpen, open, close] = useTrigger(false);
+
+  document.addEventListener('click', () => {
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:33]: put everything in useEffect and unsubscribe when the component is unmounted

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    if (isOpen) {
+      close();
+    }
+  });
+
+  const dropdownListener = (e: React.MouseEvent) => {
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:39]: event: React.MouseEvent

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+      <div>
+        {selectedLanguages.length === 0 && (
+          <span css={styles.placeholder}>{placeholder}</span>
+        )}
+        {selectedLanguages.map((language) => {
+          const selectedLanguage = allLanguages.find(
+            (languageWithDescription) =>
+              languageWithDescription.value === language,
+          );
+          const removeLanguage = (e) => {
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:61]: move the function to the top of the component

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+              key={language}
+            >
+              {selectedLanguage?.label}
+              <button
+                type="button"
+                className="dif"
+                css={styles.closeIcon}
+                onClick={removeLanguage}
+              >
+                <IconMd
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:79]: use new icon

File: app/containers/EditCommunity/Form.jsx [hunk 5]
```diff
@@ -52,87 +73,176 @@ const EditCommunityForm = ({
...
-  return (
-    <ExtendedBase>
-      <Field
-        name={COMM_AVATAR_FIELD}
-        component={AvatarField}
-        validate={[imageValidation, required]}
-        warn={[imageValidation, required]}
-        disabled={communityLoading}
-      />
+  const setGeneralTab = (e) => {
```
Comment (Lampardks) [inline app/containers/EditCommunity/Form.jsx:97]: event

File: app/containers/EditCommunity/Form.jsx [hunk 6]
```diff
@@ -52,87 +73,176 @@ const EditCommunityForm = ({
...
+    e.preventDefault();
+    setTab(GENERAL_TAB);
+  };
+  const setTranslationsTab = (e) => {
```
Comment (Lampardks) [inline app/containers/EditCommunity/Form.jsx:101]: event

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 2 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx:39]: done

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 3 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx:61]: done

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 2 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx:39]: done

File: app/containers/EditCommunity/Form.jsx (hunk 6 above)
Comment (AweAwe95) [reply app/containers/EditCommunity/Form.jsx:101]: done

File: app/containers/EditCommunity/Form.jsx (hunk 5 above)
Comment (AweAwe95) [reply app/containers/EditCommunity/Form.jsx:97]: done

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 1 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx:33]: done

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+    event.stopPropagation();
+    if (selectedLanguages.length !== allLanguages.length) {
+      isOpen ? close() : open();
+    }
+  };
+
+  const removeLanguage = (event: React.MouseEvent, language: string) => {
+    event.preventDefault();
+    removeSelectedLanguage(language);
+  };
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:53]:
```
const removeLanguage = (language: string) => (event: React.MouseEvent) => {
    event.preventDefault();
    removeSelectedLanguage(language);
  };
```

File: app/components/CommunitySelector/DropdownTrigger.tsx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+              className="dif aic jcc fz14"
+              css={styles.language}
+              key={language}
+            >
+              {selectedLanguage?.label}
+              <button
+                type="button"
+                className="dif"
+                css={styles.closeIcon}
+                onClick={(event) => removeLanguage(event, language)}
```
Comment (Lampardks) [inline app/components/CommunitySelector/DropdownTrigger.tsx:82]:
```
removeLanguage(language)
```

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 7 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx:53]: done

File: app/components/CommunitySelector/DropdownTrigger.tsx (hunk 8 above)
Comment (AweAwe95) [reply app/components/CommunitySelector/DropdownTrigger.tsx]: done

**Review by Lampardks — APPROVED** [review]

**Review by YuriyLu — APPROVED** [review]

**Pull request closed without merging**